jazz-ui/src/app/secondary-components/create-service/oss/create-service.component.html

```diff
@@ -65,6 +65,30 @@ <h4 class="bold-title">
                 </div>
             </div>
         </section>
+        <section class="each-step-wrap api-deployment" *ngIf="typeOfService === 'api'">
```

Comment:
This must be configuration driven. This should be enabled only when apigee component is enabled by Jazz admin.